src/components/textAreas/textAreaRichText/textAreaRichTextActions.test.tsx

```diff
@@ -0,0 +1,124 @@
+import { fireEvent, render, screen } from '@testing-library/react';
```

Comment:
Recently found out about this new companion library that Kent recomends over using the `fireEvent` functionality: https://testing-library.com/docs/user-event/intro/.

Took a look at the article about why/when to choose one over the other, and it's an interesting read.

Reply:
I've used `userEvent` before as well, it gives a better abstraction when firing events. We might consider following his suggestion of always using `userEvent` whenever possible and fallback to `fireEvent` only when the event is not supported by the first. Will create a ticket for this 👌🏻